addon/serializers/web-api.js

```diff
@@ -10,10 +11,10 @@ export default DS.RESTSerializer.extend({
 
     if(isCollection) {
       payload.forEach((item) => {
-        this.extractRelationships(store, payloadWithRoot, item, primaryType);
+        this.privateExtractRelationships(store, payloadWithRoot, item, primaryType);
```

Comment:
1. yes, it broke lots of stuff when using relationships :)
2. sure, i'll change it.